test/port_helpers.py

```diff
@@ -108,5 +111,6 @@ def bind_port(sock, host=HOST):
             sock.setsockopt(socket.SOL_SOCKET, socket.SO_EXCLUSIVEADDRUSE, 1)
 
     sock.bind((host, 0))
-    port = sock.getsockname()[1]
+    port: int = sock.getsockname()[1]
```

Comment:
So now the `: int` should not be necessary so I suggest removing it.

Reply:
ah, sorry!